applications/vanilla/controllers/class.discussionscontroller.php

```diff
@@ -131,10 +136,13 @@ public function index($Page = false) {
                 null,
                 Gdn::request()->get('save')
             );
+            if (in_array($this->SelfUrl, $this->categoryFilterList)) {
```

Comment:
`// We need to check against a whitelist due to addons calling this method ad hoc.`